go/signedexchange/certurl/certchain.go

```diff
@@ -12,13 +15,13 @@ import (
 	"github.com/WICG/webpackage/go/signedexchange/cbor"
 )
 
-type CertChainItem struct {
+type AugmentedCertificate struct {
```

Comment:
Would you describe what `AugmentedCertificate` is as a comment?

Reply:
(Would you add a specref?)

Reply:
Spec ref added.